dev-docs/bidders/nativo.md

```diff
@@ -4,12 +4,15 @@ title: Nativo
 description: Prebid Nativo Bidder Adapter
 pbjs: true
 pbs: true
+media_types: banner, native, video
```

Comment:
If you support multiple media types, please also add

```
multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
```

Reply:
Thanks @muuki88  - added